Add mailing lists (#714)
publiccodenet/about: activities/tool-management/mailing-lists.md

---
type: resource, tooling
---

# Mailman mailing lists

> Mailman is free software for managing electronic mail discussion and e-newsletter lists. We use Mailman because it supports built-in archiving, automatic bounce processing, content filtering, digest delivery, spam filters, and because it's open-source.

Reviewer: ```suggestion
Mailman is free software for managing electronic mail discussion and e-newsletter lists. We use Mailman because it supports built-in archiving, automatic bounce processing, content filtering, digest delivery and spam filters, and because it's open-source.
```

## 1. Guidance for people who join our mailing lists

### How to join

To subscribe to the mailing list 'listname' you can send an email with 'subscribe' in the subject to 'listname-join@lists.publiccode.net'. By subscribing to our lists you give us consent to use your email address for mailing list purposes only.

You can also subscribe at the URL and [select the mailing list you want to be part of](https://lists.publiccode.net/mailman/postorius/lists/?all-lists).

You will receive an email to confirm your email address registration. Confirm your email address by replying to the message keeping the subject header intact. If you change your mind, you can simply ignore the message.

After confirming the email address another mail will be sent:

> > Welcome to the 'listname' mailing list!

Reviewer 2: A blockquote needs to be surrounded by blank lines.

```suggestion

> Welcome to the 'listname' mailing list!
```

If you haven't received the welcome message please make sure to check your spam folder!

For general information on using the Mailman mailing lists you can have a look at the [user documentation](https://wiki.list.org/DOC/Mailman%202.1%20Members%20Manual).

### What happens with your data

The system registers contact details (name, e-mail) for use in the administration of the subscription to lists and forums in Mailman. We will delete the contact details when the user terminates the subscription.

### How to leave

The easiest way to unsubscribe from the mailing list is to send a message to listname-leave@lists.publiccode.net. Mailman will send you a confirmation message. If you reply to that message you will be unsubscribed. You do not need to know your password to do this.

You can also unsubscribe via the web page by [simply clicking on the mailing list you are on]( https://lists.publiccode.net/mailman/postorius/lists/?all-lists) and then clicking "unsubscribe".

## Guidance for mailing list administrators

> 

Reviewer: ```suggestion
You should make sure potential and current mailing list members have easy access to the guidance for people who join our mailing lists.
```

> You should make sure potential and current mailing list members have easy access to the guidance for people who join our mailing lists.

Reviewer: ```suggestion

You should make sure potential and current mailing list members have easy access to the guidance for people who join our mailing lists.

```

---
type: resource, tooling
---

# Mailman mailing lists

Mailman is free software for managing electronic mail discussion and e-newsletter lists. We use Mailman because supports built-in archiving, automatic bounce processing, content filtering, digest delivery, spam filters, and because it's open-source.

## 1. Guidance for people who join our mailing lists

* How to join

To subscribe to the mailing list 'listname' you can send an email with 'subscribe' in the subject to 'listname-join@lists.publiccode.net'. By subscribing to our lists you give us consent to use your email address for mailing list purposes only.

You can also subscribe at the URL and [select the mailing list you want to be part of](https://lists.publiccode.net/mailman/postorius/lists/?all-lists).

You will receive an email to confirm your email address registration. Confirm your email address by replying to the message keeping the subject header intact. If you change your mind, you can simply ignore the message.

After confirming the email address another mail will be sent:

> >

Reviewer 2: The quote needs top be surrounded by blank lines.

```suggestion

```

You can also unsubscribe via the web page by [simply clicking on the mailing list you are on]( https://lists.publiccode.net/mailman/postorius/lists/?all-lists) and then clicking "unsubscribe".

## 2. Guidance for mailing list administrators

* What to do with people's data

When a user signs up to our mailing list they give us consent to use that email address for mailings as per [our privacy policy](https://about.publiccode.net/organization/privacy.html). When the user unsubscribes we will remove the email address from the database.

* How we handle code of conduct violations

In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.

> By default, the full [Foundation for Public Code code of conduct](https://about.publiccode.net/CODE_OF_CONDUCT.html) applies to all of our mailing lists. Some of the codebases in our stewardship have other codes of conduct, which will be linked from the description page of that mailing list.

Reviewer 2: This file is in the same repository.

```suggestion
By default, the full [Foundation for Public Code code of conduct](../../CODE_OF_CONDUCT.md) applies to all of our mailing lists. Some of the codebases in our stewardship have other codes of conduct, which will be linked from the description page of that mailing list.
```
